iot-e2e-tests/android/readme.md

```diff
@@ -0,0 +1,18 @@
+# Microsoft Azure IoT Hub Java Android tests
```

Comment:
There is one more request here. Turns out we need a script for all of these installations. Can you please write one for these ?